### PR TITLE
Use symlink for node_modules

### PR DESCRIPTION
**Pull request by mjethani**
Using a symlink for `node_modules`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,7 +51,7 @@ update-submodules:
 	tools/update-submodules.sh
 
 clean:
-	rm -rf dist/build
+	rm -rf dist/build tmp
```
Comment (gorhill) [inline Makefile:54]: We can't remove `tmp` though, I have been using it to put ubo-related stuff for years now -- maybe just remove `tmp/node_modules`?

**Comment by mjethani** [comment]
@gorhill 

```
cd dist/build/uBlock0.npm
npm install
find node_modules/* -type f | xargs sha1sum -b | cut -f1 -d' ' | sha1sum | cut -f1 -d' '
```

Are you getting `fee1f04594e13b86a4ae3d3a9b43a865ef59d6ba`?

File: Makefile (hunk 1 above)
Comment (mjethani) [reply Makefile]: Done.

**Comment by gorhill** [comment]
> Are you getting `fee1f04594e13b86a4ae3d3a9b43a865ef59d6ba`?

I am getting `4e897b5fb800568120c799cd6052a17ed2c762b3`.

**Comment by mjethani** [comment]
> > Are you getting `fee1f04594e13b86a4ae3d3a9b43a865ef59d6ba`?
> 
> I am getting `4e897b5fb800568120c799cd6052a17ed2c762b3`.

I see.

I was thinking of verifying that SHA-1 digest in addition to the checks `npm` already does, but I guess the value is different on different systems.

**Pull request merged**